### PR TITLE
W3DOverlordAircraftDraw

### PR DESCRIPTION
**Pull request by jonwil**

### CODE REVIEW

File: src/game/client/draw/w3doverlordaircraftdraw.cpp [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+void W3DOverlordAircraftDraw::Do_Draw_Module(const Matrix3D *transform)
+{
+    W3DModelDraw::Do_Draw_Module(transform);
+    Object *object = Get_Drawable()->Get_Object();
+
+    if (object != nullptr && object->Get_Contain() != nullptr) {
+        const Object *rider = object->Get_Contain()->Friend_Get_Rider();
+
+        if (rider != nullptr) {
+            captainslog_dbgassert(rider->Get_Drawable() != nullptr, "OverlordAircraftDraw finds no rider's drawable");
```
Comment (xezon) [inline src/game/client/draw/w3doverlordaircraftdraw.cpp]: Please simplify this code to improve readability.
```c++
Drawable *drawable = rider->Get_Drawable();
if (drawable != nullptr) {
...
} else {
    captainslog_dbgassert(0, "OverlordAircraftDraw finds no rider's drawable");
}
```

File: src/game/client/draw/w3doverlordaircraftdraw.cpp [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+}
+
+void W3DOverlordAircraftDraw::CRC_Snapshot(Xfer *xfer)
+{
+    W3DModelDraw::CRC_Snapshot(xfer);
+}
+
+void W3DOverlordAircraftDraw::Xfer_Snapshot(Xfer *xfer)
+{
+    unsigned char ver = 1;
```
Comment (xezon) [inline src/game/client/draw/w3doverlordaircraftdraw.cpp]: `uint8_t`

File: src/game/client/draw/w3doverlordaircraftdraw.cpp (hunk 1 above)
Comment (jonwil) [reply src/game/client/draw/w3doverlordaircraftdraw.cpp]: fixed.

File: src/game/client/draw/w3doverlordaircraftdraw.cpp (hunk 2 above)
Comment (jonwil) [reply src/game/client/draw/w3doverlordaircraftdraw.cpp]: fixed.

**Comment by xezon** [comment]
Code looks plausible in principle. Helix helicopter looks normal in game.

**Review by xezon — APPROVED** [review]

**Review by tomsons26 — APPROVED** [review]

**Pull request merged**